tietokanta/src/main/resources/db/migration/V1_1081__.sql

```diff
@@ -0,0 +1,128 @@
+ALTER TABLE tarkastus_2016_q3 ADD CONSTRAINT tarkastus_2016_q3_pkey PRIMARY KEY USING INDEX tarkastus_2016_q3_id_idx;
+ALTER TABLE tarkastus_2016_q4 ADD CONSTRAINT tarkastus_2016_q4_pkey PRIMARY KEY USING INDEX tarkastus_2016_q4_id_idx;
+ALTER TABLE tarkastus_2017_q1 ADD CONSTRAINT tarkastus_2017_q1_pkey PRIMARY KEY USING INDEX tarkastus_2017_q1_id_idx;
+ALTER TABLE tarkastus_2017_q2 ADD CONSTRAINT tarkastus_2017_q2_pkey PRIMARY KEY USING INDEX tarkastus_2017_q2_id_idx;
+ALTER TABLE tarkastus_2017_q3 ADD CONSTRAINT tarkastus_2017_q3_pkey PRIMARY KEY USING INDEX tarkastus_2017_q3_id_idx;
+ALTER TABLE tarkastus_2017_q4 ADD CONSTRAINT tarkastus_2017_q4_pkey PRIMARY KEY USING INDEX tarkastus_2017_q4_id_idx;
+ALTER TABLE tarkastus_2018_q1 ADD CONSTRAINT tarkastus_2018_q1_pkey PRIMARY KEY USING INDEX tarkastus_2018_q1_id_idx;
+ALTER TABLE tarkastus_2018_q2 ADD CONSTRAINT tarkastus_2018_q2_pkey PRIMARY KEY USING INDEX tarkastus_2018_q2_id_idx;
+ALTER TABLE tarkastus_2018_q3 ADD CONSTRAINT tarkastus_2018_q3_pkey PRIMARY KEY USING INDEX tarkastus_2018_q3_id_idx;
+ALTER TABLE tarkastus_2018_q4 ADD CONSTRAINT tarkastus_2018_q4_pkey PRIMARY KEY USING INDEX tarkastus_2018_q4_id_idx;
+ALTER TABLE tarkastus_2019_q1 ADD CONSTRAINT tarkastus_2019_q1_pkey PRIMARY KEY USING INDEX tarkastus_2019_q1_id_idx;
+ALTER TABLE tarkastus_2019_q2 ADD CONSTRAINT tarkastus_2019_q2_pkey PRIMARY KEY USING INDEX tarkastus_2019_q2_id_idx;
+ALTER TABLE tarkastus_2019_q3 ADD CONSTRAINT tarkastus_2019_q3_pkey PRIMARY KEY USING INDEX tarkastus_2019_q3_id_idx;
+ALTER TABLE tarkastus_2019_q4 ADD CONSTRAINT tarkastus_2019_q4_pkey PRIMARY KEY USING INDEX tarkastus_2019_q4_id_idx;
+ALTER TABLE tarkastus_2020_q1 ADD CONSTRAINT tarkastus_2020_q1_pkey PRIMARY KEY USING INDEX tarkastus_2020_q1_id_idx;
+ALTER TABLE tarkastus_2020_q2 ADD CONSTRAINT tarkastus_2020_q2_pkey PRIMARY KEY USING INDEX tarkastus_2020_q2_id_idx;
+ALTER TABLE tarkastus_2020_q3 ADD CONSTRAINT tarkastus_2020_q3_pkey PRIMARY KEY USING INDEX tarkastus_2020_q3_id_idx;
+ALTER TABLE tarkastus_2020_q4 ADD CONSTRAINT tarkastus_2020_q4_pkey PRIMARY KEY USING INDEX tarkastus_2020_q4_id_idx;
+ALTER TABLE tarkastus_2021_q1 ADD CONSTRAINT tarkastus_2021_q1_pkey PRIMARY KEY USING INDEX tarkastus_2021_q1_id_idx;
+ALTER TABLE tarkastus_2021_q2 ADD CONSTRAINT tarkastus_2021_q2_pkey PRIMARY KEY USING INDEX tarkastus_2021_q2_id_idx;
+ALTER TABLE tarkastus_2021_q3 ADD CONSTRAINT tarkastus_2021_q3_pkey PRIMARY KEY USING INDEX tarkastus_2021_q3_id_idx;
+ALTER TABLE tarkastus_2021_q4 ADD CONSTRAINT tarkastus_2021_q4_pkey PRIMARY KEY USING INDEX tarkastus_2021_q4_id_idx;
+ALTER TABLE tarkastus_2022_q1 ADD CONSTRAINT tarkastus_2022_q1_pkey PRIMARY KEY USING INDEX tarkastus_2022_q1_id_idx;
+ALTER TABLE tarkastus_2022_q2 ADD CONSTRAINT tarkastus_2022_q2_pkey PRIMARY KEY USING INDEX tarkastus_2022_q2_id_idx;
+ALTER TABLE tarkastus_2022_q3 ADD CONSTRAINT tarkastus_2022_q3_pkey PRIMARY KEY USING INDEX tarkastus_2022_q3_id_idx;
```

Comment:
Puuttuuko 2022 q4?

Reply:
Ja 2023 q1

Reply:
Puuttuihan ne, mikä on outoa, kun tein homman IDEA:n taululistasta copy pastena.. Noh, lisäilen ja kävin vielä käsipelillä kaiken läpi. Kohta ei pitäis puuttua mitään.